test/e2e/specs/editor/various/copy-cut-paste.spec.js

```diff
@@ -450,7 +450,7 @@ test.describe( 'Copy/cut/paste', () => {
 		// back to default browser behaviour, allowing the browser to insert
 		// unfiltered HTML. When we swap out the post title in the post editor
 		// with the proper block, this test can be removed.
-		pageUtils.setClipboardData( {
+		await pageUtils.setClipboardData( {
```

Comment:
I'm seeing `Unexpected `await` of a non-Promise (non-"Thenable") value.` ESLint errors here.

Reply:
This is possibly due to outdated build. Try running `npm run dev` or `npm run build` again and see if eslint still errors?